docs/database/supplementary-guidelines/migration-guide.md

```diff
@@ -0,0 +1,249 @@
+| `db.name`                                           | Removed, integrated into the new `db.namespace`                                                             |
+| `db.redis.database_index`                           | Removed, integrated into the new `db.namespace`                                                             |
+| `db.mssql.instance_name`                            | Removed, integrated into the new `db.namespace`                                                             |
+| `db.instance.id`                                    | Removed, replaced by `server.address` or integrated into `db.namespace` as appropriate                      |
+| `db.statement` &rarr; `db.query.text`               | Clarified, SHOULD be collected by default only if there is sanitization that excludes sensitive information |
+| `db.operation` &rarr; `db.operation.name`           |                                                                                                             |
+| `db.sql.table` &rarr; `db.collection.name`          |                                                                                                             |
+| `db.cassandra.table` &rarr; `db.collection.name`    |                                                                                                             |
+| `db.mongodb.collection` &rarr; `db.collection.name` |                                                                                                             |
+| `db.cosmosdb.container` &rarr; `db.collection.name` |                                                                                                             |
+| New: `db.query.parameter.<key>`                     | Opt-In                                                                                                      |
+| New: `error.type`                                   |                                                                                                             |
+<!-- prettier-ignore-end -->
+
+References:
+
+- [Database client span attributes v1.24.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/database/database-spans.md)
+- [Database client span attributes v1.TODO (stable)](https://github.com/open-telemetry/semantic-conventions/blob/v1.TODO/docs/database/database-spans.md)
+
+### Database client span names
+
+The recommended span name has changed to `{db.operation.name} {target}`, where the `{target}` SHOULD describe the entity
+that the operation is performed against and SHOULD adhere to one of the following values, provided they are accessible:
+
+- `db.collection.name` SHOULD be used for data manipulation operations or operations on database collections.
+- `db.namespace` SHOULD be used for operations on a specific database namespace.
+- `server.address:server.port` SHOULD be used for other operations not targeting any specific database(s) or collection(s)
```

Comment:
tiny nit: add a period at the end of this sentence or remove the ones from the bullet points above to keep consistent